<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ThirdPluginDlg</class>
 <widget class="QWidget" name="ThirdPluginDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>752</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>第三方插件管理</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#ThirdPluginDlg
{
	background-color:rgb(240,240,240);
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frmPlugList">
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="styleSheet">
         <string notr="true">QLabel
{
	font-family:&quot;Microsoft Yahei&quot;;
	font-size:13px;
	background-color:rgb(88,148,255);
	color:white;
	min-height:30px;
	padding-left:10px;
}</string>
        </property>
        <property name="text">
         <string>插件列表</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frameThirdPluginToolBar">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>25</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QFrame#frameThirdPluginToolBar
{
	background-color:#eef0ff;
	color:black;;
	border-width:0px;
	border-style:solid;
	border-color:#cccccc;
	border-left-width:1px;
	border-right-width:1px;
}</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QToolButton" name="btnAddThirdPlugin">
           <property name="styleSheet">
            <string notr="true">QToolButton
{
	border:none;
	font-family:&quot;Microsoft Yahei&quot;;
	background-color:transparent;
}

QToolButton:hover
{
	background-color:#d5d5d5;
}

QToolButton:pressed
{
	background-color:#bbbbbb;
}

QToolButton:disabled
{
	background-color:transparent;
}</string>
           </property>
           <property name="text">
            <string>新增</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="btnUpdateThirdPlugin">
           <property name="styleSheet">
            <string notr="true">QToolButton
{
	border:none;
	font-family:&quot;Microsoft Yahei&quot;;
	background-color:transparent;
}

QToolButton:hover
{
	background-color:#d5d5d5;
}

QToolButton:pressed
{
	background-color:#bbbbbb;
}

QToolButton:disabled
{
	background-color:transparent;
}</string>
           </property>
           <property name="text">
            <string>更新</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="btnDeleteThirdPlugin">
           <property name="styleSheet">
            <string notr="true">QToolButton
{
	border:none;
	font-family:&quot;Microsoft Yahei&quot;;
	background-color:transparent;
}

QToolButton:hover
{
	background-color:#d5d5d5;
}

QToolButton:pressed
{
	background-color:#bbbbbb;
}

QToolButton:disabled
{
	background-color:transparent;
}</string>
           </property>
           <property name="text">
            <string>删除</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="tbThirdPluginList">
        <property name="styleSheet">
         <string notr="true">
QHeaderView::section 
{
	background-color:white;
	color:black;
	min-height:25px;
	border-width:0px;
	border-style:solid;
	border-color:#cccccc;
	border-right-width:1px;
	border-bottom-width:1px;
}

QTableWidget:item
{
	color:black;
	padding-left:10px;
}

 QHeaderView::down-arrow {
     image: none;
 }

 QHeaderView::up-arrow {
     image:none;
}
QWidget#tbThirdPluginList
{
	border:1px solid #cccccc;
}</string>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="showGrid">
         <bool>false</bool>
        </property>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>50</number>
        </attribute>
        <attribute name="horizontalHeaderHighlightSections">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderDefaultSectionSize">
         <number>25</number>
        </attribute>
        <attribute name="verticalHeaderHighlightSections">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string>  插件名</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>  版本号</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">QLabel
{
	font-family:&quot;Microsoft Yahei&quot;;
	font-size:13px;
	background-color:rgb(88,148,255);
	color:white;
	min-height:30px;
	padding-left:10px;
}</string>
       </property>
       <property name="text">
        <string>文件列表</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTreeWidget" name="trThirdPluginFileList">
       <property name="styleSheet">
        <string notr="true">/* QHeaderView::section {
	background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(172, 197, 224, 255), stop:1 rgba(200, 230, 255, 225));
    padding-left: 4px;
	border:none;
	border-right: 1px solid qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(130, 140, 190, 255), stop:1 rgba(172, 197, 255, 225));;
	height:25px;
	font-family:&quot;微软雅黑&quot;;
 }*/

QHeaderView 
{
	font-family:&quot;微软雅黑&quot;;
	background-color:#eef0ff;
	color:black;
}
QHeaderView::section 
{
	background-color:#eef0ff;
	color:black;
	min-height:25px;
	border-width:0px;
	border-style:solid;
	padding-left:5px;
	border-color:#cccccc;
	border-bottom-width:1px;
	border-top-width:1px;
}

QTableWidget:item
{
	padding-left:10px;
}

QFrame#trThirdPluginFileList{
	border:1px solid #cccccc;
	border-top-width:0px;
}

 QHeaderView::down-arrow {
     image: none;
 }

 QHeaderView::up-arrow {
     image:none;
}

 QScrollBar:vertical {
     border: 0px solid grey;
     background: transparent;
     width: 10px;
 }
 QScrollBar::handle:vertical {
     background: #cccccc;;
     min-height: 100px;
 }
 QScrollBar::handle:vertical:hover {
     background: #aaaaaa;
     min-heiht: 100px;
 }
 QScrollBar::add-line:vertical {
	border:none;
	background:transparent;
 }
 QScrollBar::sub-line:vertical {
	border:none;
	background:transparent;
 }

 QScrollBar:horizontal {
     border: 0px solid grey;
     background: transparent;
     height: 5px;
 }
 QScrollBar::handle:horizontal {
     background: #cccccc;;
     min-width: 100px;
 }
 QScrollBar::handle:horizontal:hover {
     background: #aaaaaa;
     min-width: 100px;
 }
 QScrollBar::add-line:horizontal {
	border:none;
	background:transparent;
 }
 QScrollBar::sub-line:horizontal {
	border:none;
	background:transparent;
 }</string>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="indentation">
        <number>0</number>
       </property>
       <attribute name="headerDefaultSectionSize">
        <number>200</number>
       </attribute>
       <attribute name="headerMinimumSectionSize">
        <number>30</number>
       </attribute>
       <column>
        <property name="text">
         <string>名称</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>修改日期</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="styleSheet">
        <string notr="true">QLabel
{
	font-family:&quot;Microsoft Yahei&quot;;
	font-size:13px;
	background-color:rgb(88,148,255);
	color:white;
	min-height:30px;
	padding-left:10px;
}</string>
       </property>
       <property name="text">
        <string>文件预览</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="edtPreview">
       <property name="styleSheet">
        <string notr="true">QWidget
{
	border: 1px solid #cccccc;
}</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
